jib-cli/src/main/java/com/google/cloud/tools/jib/cli/CommonCliOptions.java

```diff
@@ -244,6 +244,15 @@
   @SuppressWarnings("NullAway.Init") // initialized by picocli
   private boolean serialize;
 
+  @CommandLine.Option(
+      names = "--image-json",
```

Comment:
What do you think of `--result-json`?

Reply:
That's a good idea. Hm I'm looking for something that describes the function of the json. how about `--image-metadata`?